source/transfer_list.rst

```diff
@@ -994,6 +995,177 @@ into its memory map during platform setup. If other memory types are required
      - hdr_size + 0x8
      - The size of the memory region.
 
+.. _opendice_entries:
+
+Entries related to Open DICE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These entries contain [Open-DICE]_ information elements that must be transferred between boot stages in a DICE architecture where a DICE Protection Environment (DPE) is not used.
+
+This includes the Compound Device Identifier (CDI) used for attestation and the CDI used for sealing, as well as the accumulated certificate chain.
+
+.. note::
+  As described in [DICE]_, CDIs are very sensitive data and are as long-lived as the layers they represent.
+  Each layer is therefore responsible for the proper handling of the CDIs it owns, as well as the derived cryptographic key material.
+  In particular, any given layer is expected to use and erase the CDI it received.
+  While the CDI is used, the layer needs the ability to prevent access to the CDI and disclosure of the value.
+
+The following entry types are currently defined:
+
+**CDI for attestation (XFERLIST_OPENDICE_CDI_ATTEST)**
+
+This entry type holds the "Attestation CDI" described in [Open-DICE]_.
+
+The CDI computed by the Sender's boot stage SHALL be added to the TL.
+The CDI received from the previous boot stage (i.e., DICE layer) MUST NOT be forwarded.
```

Comment:
> I think we need a comment here about what the receiver should do with the profile_id, e.g. use this to generate a CDI that is compatible with the previous layer's CDI as per the Android OpenDICE spec.

What is the behaviour in case $L_{i+1}$ does not implement a profile compatible with layer $L_i$?